### PR TITLE
Return Ukrainian translation to the state before it was sabotaged by "Danilo Negrilo <gavrilonegrilo@tutanota.com>"

### PR DESCRIPTION
**Pull request by AustinPowers1935**
Revert "fix: remove insulting l10n strings"
This reverts commit 01b9616a7472a08434ead9e046e9f9b02514ab8a.

Revert "l10n: translated using Weblate (Ukrainian)" This reverts commit 6f4028057e55cebfc53cc45cb39831f7e6a176cb.

Revert "l10n: translated using Weblate (Ukrainian)" This reverts commit 29069dbbd345b76ca75d0977952cd4716d414f85.

Revert "l10n: translated using Weblate (Ukrainian)" This reverts commit 3c4b6c25b17ff581c3499a4136d917ac2cf20889.

Revert "l10n: translated using Weblate (Ukrainian)" This reverts commit c75736c198b465fa21c63b33dd058d8fd1f4980a.

Revert "l10n: translated using Weblate (Ukrainian)" This reverts commit 6659362baacdb14a5e33a7f3d6f1dfbfd91c3da9.

### CODE REVIEW

**Comment by AustinPowers1935** [comment]
@d-loose You removed valid translations and did not remove part of malicious. Proper fix: revert all commits made by user "Danilo Negrilo <gavrilonegrilo@tutanota.com>" and ofc ban it instead of removing Ukrainian translation. 
Also by removing Ukrainian translation you helped this user achieve its intentions.

**Comment by AustinPowers1935** [comment]
@d-loose Also I would like to add additional observation about the name of this user.
First part of name "Danilo" seems like was intended to sound like Ukrainian "Данило"[Danylo] (russian form is "Данил"[Danil]). But in russian and Ukrainian the letter "и" sounds differently therefore it failed to make proper transliteration: it used russian "и" (Ukrainian "i", sounds as "ee" in "deed") instead of Ukrainian "и" (sounds kind of like "i" in "this") so from Ukrainian it should be transliterated as "Danylo". 
Second part of its name is some form of n-word and also it should be transliterated differently if it was supposed to be Ukrainian instead of russian (the same issue with "и").

File: packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_uk.arb [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+  },
+  "rstTitle": "Вимкнути RST",
+  "@rstTitle": {},
+  "rstDescription": "Цей комп’ютер використовує технологію Intel RST (Rapid Storage Technology). Перед встановленням Ubuntu вам потрібно вимкнути RST у Windows.",
+  "@rstDescription": {},
+  "quitButtonText": "Скасувати встановлення",
+  "@quitButtonText": {},
+  "configureSecureBootTitle": "Налаштувати безпечне завантаження",
+  "@configureSecureBootTitle": {},
+  "configureSecureBootDescription": "Ви вирішили встановити стороннє програмне забезпечення для драйверів. Для цього необхідно вимкнути безпечне завантаження.\nДля цього потрібно вибрати ключ безпеки зараз, та ввести його після перезавантаженні системи.",
```
Comment (BorysCS) [inline packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_uk.arb:65]: перезавантаження*

**Review by delgod — APPROVED** [review]

**Comment by tim-hm** [comment]
@AustinPowers1935 thanks for this. We pulled the translations in an initial abundance of caution as we (continue to) assess the impact. We intend to have correct Ukrainian translations back into a new daily ISO later today.

**Comment by Saltane** [comment]
Ubuntu seems to be not the first victim of this person's "contributions", there's another project that has also acquired similar "translations". Whoever the person is behind Danilo's account, it is apparent that they suffer from severe lack of touching grass in organism.
It's a shame that case like this is a thing today, right when Ukraine is experiencing biggest existential threat in its independent history.

**Comment by seb128** [comment]
ideally the fixed strings should also be pushed to weblate

**Comment by inobelar** [comment]
Thanks for making this PR to cleanup vandalism!

As I can see from [changes](https://github.com/canonical/ubuntu-desktop-provision/commit/896aa262382aa3f9ccd27994fbcbc7fcb6242b84#diff-b7eb74e06861692e751cdc1640f0db5b1923277f58af1f0e0b6fb40d60b6fe66)

image

Currently (in master branch) damaged the next files:
1. [packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_localizations_uk.dart](https://github.com/canonical/ubuntu-desktop-provision/blob/main/packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_localizations_uk.dart)
2. [packages/ubuntu_init/lib/src/l10n/ubuntu_init_localizations_uk.dart](https://github.com/canonical/ubuntu-desktop-provision/blob/main/packages/ubuntu_init/lib/src/l10n/ubuntu_init_localizations_uk.dart)
3. [packages/ubuntu_init/lib/src/l10n/ubuntu_init_uk.arb](https://github.com/canonical/ubuntu-desktop-provision/blob/main/packages/ubuntu_init/lib/src/l10n/ubuntu_init_uk.arb) (fixes for it spotted in: https://github.com/canonical/ubuntu-desktop-provision/pull/169)
4. `packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_uk.arb` - (re-introduce in this commit)

@AustinPowers1935 maybe, to join forces in cleanup, we need to fix `1.` and `2.` files too ?

Offtop: I spotted similar vandalism in: [FreshBSD](https://freshbsd.org/hello/Menu/commit/6f4f432a9accd7627f11f70e1d49e35fc7ca34bc)

**Review by Oleksii909 — COMMENTED** [review]
It seems one more file needs to be fixed:
adding translation file from your PR is not enough - to activate the lines, it seems that you need to make changes to packages/ubuntu_bootstrap/lib/l10n/ubuntu_bootstrap_localizations_uk.dart

**Comment by d-loose** [comment]
Let's stick to removing the malicious translations in this PR and sort out the missing ones afterwards.

**Review by d-loose — APPROVED** [review]

**Review by PashaShunkow — APPROVED** [review]

**Review by svicervlad — APPROVED** [review]

**Comment by elderkeltir** [comment]
moscovites strikes back! in usual to them casually xenophobic way =\

**Comment by senchenko-oleksiy** [comment]
The malicious contributor has been identified as Semyon Ryndin, a Russian national working at Canonical’s offices in San Francisco. According to Canonical, he was promptly fired following the incident and faces risk of deportation.

**Comment by netroy** [comment]
> Semyon Ryndin, a Russian national

If you search for that name, you won't find any news or a person by that name, however you will find this exact same comment being posted on various other forums.
This comment is likely disinformation, and should be removed.

**Comment by webknjaz** [comment]
> This comment is likely disinformation, and should be removed.

Yes, I tried to fact-check that, and it comes out empty. Plus a trusted contact at Canonical said that they don't actually know who that vandal is. There's two fake accounts on Weblate. One created a month ago, the other one — yesterday. Both have an incorrect transliteration of the Ukrainian name at the beginning of the username followed offensive references widespread among r*ssians. The translation strings are obviously software-produced and stylistically match typical fakes made by the ruscists, with silly lexical and grammar mistakes a native Ukrainian speaker would never do.

While the specific piece of suggestion is fake (coming from an anonymous account on 4chan of all places), that altogether only has one realistic explanation — the ruscists are behind this. And it's quite obvious to those in the context. And it did have the desired effect — under the abusive commit, a mob of the ruscist accounts materialized quite quickly, only to harass Ukrainians and post that speaking Ukrainian implies being a nazi unconditionally 🤡.
This is another bit of behavior we routinely see from people of r*ssian mindset/culture (not necessarily geography).

That said, removing it is probably unnecessary as it's important for demonstration purposes to show people how actual propaganda works.

**Comment by tim-hm** [comment]
I've removed that comment and reported it. I'll lock this thread as it is not merged.

**Pull request merged**